### ISSUE TITLE
The REPL shows an error when trying to send contains(aNumber) to a collection of objects

### ISSUE THREAD

**Issue by gcrespi** [uqbar-project/wollok#532]
![captura de pantalla de 2015-12-26 14 32 43](https://cloud.githubusercontent.com/assets/10291211/12007319/8e9e3500-abdd-11e5-831f-1f0359f89206.png)

**Comment by javierfernandes** [comment]
The problem seems to happen when sending a number as argument to the method.
Because this works fine
```xtend
object a {}
		object b {}
		object c {}
		object d {}
		program p {
			val l = [a, b, c]
			assert.that(l.contains(a))
			assert.that(l.contains(b))
			assert.that(l.contains(c))
			assert.notThat(l.contains(d))
		}
```

But this fails
```xtend
assert.notThat(l.contains(4))
```

**Comment by javierfernandes** [comment]
Fixed. The problem was actually #571.
Plus the fact that the exception was not being correctly shown, also fixed now #570
